check_sites.py

```diff
@@ -12,12 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from RwsCheck import RwsCheck
+import difflib
 import json
 import getopt
 import sys
 import os
 from publicsuffix2 import PublicSuffixList
 
+def load_rws_file_as_json(input_file, with_format):
```

Comment:
Nit: preexisting problem, but `input_file` is ambiguous - is that the name of the file or the contents of it? IMO we should call this `input_filename` or something.